saleor/static/dashboard-next/orders/components/OrderCustomer/OrderCustomer.tsx

```diff
@@ -93,17 +91,12 @@ const OrderCustomer = decorate<OrderCustomerProps>(
           <Typography>{i18n.t("Anonymous customer")}</Typography>
         ) : (
           <>
-            <Typography>{client.email}</Typography>
-            <Typography
+            <a
+              href={`mailto:${client.email}`}
               className={classes.link}
-              onClick={
-                onCustomerEmailClick
-                  ? onCustomerEmailClick(client.id)
-                  : undefined
-              }
             >
-              {client.email}
-            </Typography>
+              <Typography>{client.email}</Typography>
```

Comment:
Can we refactor it to `<Typography component="a" href="..." />`?
I feel like it should be done that way.

Reply:
Couldn't add href prop to <Typography />.
Created <ExternalLink />.